src/Evolution/Systems/RelativisticEuler/Valencia/Characteristics.hpp

```diff
@@ -74,8 +75,131 @@ std::array<DataVector, Dim + 2> characteristic_speeds(
     const tnsr::i<DataVector, Dim>& normal) noexcept;
 // @}
 
-namespace Tags {
+/*!
+ * \brief Right eigenvectors of the Valencia formulation
+ *
+ * The right eigenvectors correspond to the columns of the matrix
+ * \f$\mathbf R = [R_-\, R_1\, R_2\, R_3\, R_+]\f$, where
```

Comment:
For most users of the code, how this matrix is to be used will be more important than the precise expressions. I suggest that before giving the full mathematical expressions, you document the usage of "char = left * cons" and "cons = right * char", being clear about the ordering of the cons and char variables.